src/varcorr.jl

```diff
@@ -1,11 +1,11 @@
 """
-VarCorr
+    VarCorr
 
 Information from the fitted random-effects variance-covariance matrices.
 
 # Members
 * `σρ`: a `NamedTuple` of `NamedTuple`s as returned from `σρs`
-* `s`: the estimate of the scale parameter in the distribution of the conditional dist'n of Y
+* `s`: the estimate of the per-observation dispersion parameter
```

Comment:
This is a slightly better formulation for GLMM, but I'm not sure which would be clearer to a normal user. Dispersion vs. scale is a toss-up for me; conditional distribution of Y is quite clear to me, but perhaps less to a normal user. What about "observation-level" instead of "per-observation"?

Reply:
"observation-level" would be fine.  I used "dispersion" because that is the name used in the `GLM` package.

Reply:
Then let's do "observation-level dispersion".

Reply:
actually, having just searched through the project ... "per-observation" is already used extensively in the docs, so let's just leave it alone.